{{cookiecutter.package_name}}/setup.py

```diff
@@ -19,25 +19,21 @@
     install_requirements = [
         'vivarium=={{cookiecutter.vivarium_version}}',
         'vivarium_public_health=={{cookiecutter.vivarium_public_health_version}}',
-        'vivarium_cluster_tools=={{cookiecutter.vivarium_cluster_tools_version}}',
-        'vivarium_inputs[data]=={{cookiecutter.vivarium_inputs_version}}',
-        
-        # These are pinned for internal dependencies on IHME libraries
-        'numpy<=1.15.4',
-        'tables<=3.4.0',
-        'pandas<0.25',        
-        
-        'scipy',
-        'matplotlib',
-        'seaborn',
-        'jupyter',
-        'jupyterlab',
-        'pytest',
-        'pytest-mock',
-        'pyyaml',
-        'jinja2',
+
         'click',
+        'gbd_mapping==2.1.0',
```

Comment:
Should this be grouped with the above packages, rather than below 'click'?

Reply:
Possibly. And it likely should have
{{cookiecutter.gbd_mapping_version}}
instead of the hard coded version. I'll make a list of things to review and include this.